docs/customization/index.md

```diff
@@ -130,6 +130,8 @@ exposed on all components, several components additionally support component-spe
 The value for each event key must be a function, which will be called with one argument, which is the instance of the
 component.
 
+Note that the below hooks exist for all components except `modalProduct` and `lineItem`.
```

Comment:
both `modalProduct` and `lineItem` are not an actual component (ie. not a file in the component folder), but they are offer as component under `option` so you can still directly modify its style, content...etc.

additionally, `modalProduct` actually does have a lifecycle hook, but its the same one with `product` because internally, `modalProduct` is the same component as `product` with additional modifier.